import/controls/TextSelectionToolbar.qml

```diff
@@ -67,6 +82,25 @@ Row {
 
     property var buttons: defaultButtons
 
+    property int fontSize: {
+        var size = Theme.fontSizeTiny
+
+        if (!portrait && size > Theme.fontSizeTinyBase) {
+            var string = ""
+            for (var i = 0; i < buttons.length; ++i) {
+                string = string + (buttons[i].label || "")
+            }
+
+            var requiredWidth = metrics.advanceWidth(string) + (buttons.length * (Theme.itemSizeSmall + (2 * horizontalSpacing)))
```

Comment:
Not sure I'm following the latter part. ItemSizeSmall maybe referring to icon, but the icon should be more of iconSizeMedium? Horizontal spacing is used between icon and label so it's only one time per button? 

String length concatenated vs individual parts summed might be slightly different because of kerning etc, but maybe doesn't matter that much.

Reply:
I thought it was iconSizeMedium too but that resulted in a number that was too small and the labels were all truncated which made me think I had to account for the item size instead because maybe the icon had a fixed size set, which was wrong but produced the right result. I'm still not sure what's going on there.

Twice per button is because there needs to be space between adjacent buttons too. But it's kind of irrelevant, I meant to set it to something smaller than paddingSmall but I forgot to initialize it and the spacing seemed adequate on testing perhaps because there's  a lot of padding in the graphic so I left it as 0. I could just take it out.